docs/feature-variables-and-me.md

```diff
@@ -0,0 +1,266 @@
+
+### Everything is optional
+
+If the app asks for a variable that is not specified in this particular experiment, then it gets back `nil` or `null`.
+
+It is thus imperative that the app has a reasonable default. On the other hand, this allows us to have experiments which configure only small parts of a feature.
+
+### Text resources
+
+In the example above, the title uses `getText()`. This gets a string value with `getString()`. The value is then used as a key to look up the app resource string.
+
+For example, on Android: `getText("settings-menu-item-title")` may get a string from the JSON `"app_menu_settings_title"`, which is then resolves to `R.string.app_menu_settings_title` which is then used to look up the String in the `Resources`.
+
+On iOS, `getText` uses a similar process via `LocalizedString` to look up the translated strings. You can specify the `tableName` as well as the `key` in the single value by joining it with a slash.
+
+For example, `getText("settings-menu-item-title")` may get a string from the JSON `"AppMenu/SettingsTitle"` which uses `bundle.localizedString("SettingsTitle", tableName: "AppMenu")` to look up a localized string. If the app doesn't use `tableName`, then you can omit it: e.g. `AppMenu_SettingsTitle` would look for `NSLocalizedString("AppMenu_SettingsTitle")`.
+
+If `getString()` returned a string, and the resource lookup didn't succeed, `getText()` falls back to the string.
+
+This means that you can use either pre-translated strings to try out experiments across locales, or target your experiment on a single language.
+
+> 🎛 Configuration
+>
+> Resource lookup via `Bundle` and `Context` uses the objects passed to nimbus at construction time at app-startup. In Firefox for iOS and Fenix this is `Bundle.main` and `context.applicationContext` respectively.
+
+### Image resources
+
+In the example above, the `icon` uses `getImage()` and its Android analog `getDrawable()`. This gets a string value from the JSON with `getString()` and then uses that value to look up the pre-bundled resource.
+
+For example on Android: `getDrawable("settings-menu-item-icon")` may get a string `"ic_settings"`, which is then resolved to `R.drawable.ic_settings`, which is then resolved to `context.resources.getDrawable(R.drawable.ic_settings)`.
+
+On iOS: `getImage("settings-menu-item-icon")` may get a string `"icon_photon_gear"`, which is then used to get the named `UIImage` with `UIImage(named:in:)`.
+
+## Making JSON more manageable
+
+We focused on the settings menu item in the above example, as a way of making a small enough example to reason about in this documentation, but it made for some very long variable names. The `Variables` object has itself a `getVariables(key: String)` method to make navigating the JSON more easily. This in turn allows the JSON to be organized in different ways.
```

Comment:
```suggestion
We focused on the settings menu item in the above example, as a way of making a small enough example to reason about in this documentation, but it made for some very long variable names. The `Variables` object has itself a `getVariables(key: String)` method to make navigating the JSON easier. This in turn allows the JSON to be organized in different ways.
```

Reply:
Is it possible to get an example of "organized in different ways"?

Reply:
Added a linking sentence to the example below.